src/SCRIPTS/BF/PAGES/pos_osd.lua

```diff
@@ -0,0 +1,249 @@
+labels[#labels + 1] = { t = name_cli,               x = x, y = inc.y(lineSpacing), sp = x + sp }
+labels[#labels + 1] = { t = "POS",                  x = x, y = inc.y(tableSpacing.header) }
+fields[#fields + 1] = {                             x = x, y = inc.y(tableSpacing.row), min = 0, max = 2047, vals = { 2, 3 } }
+
+x = x + tableSpacing.col
+y = yMinLim - tableSpacing.header
+
+labels[#labels + 1] = { t = "",                     x = x, y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "",                     x = x, y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "",                     x = x, y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "",                     x = x, y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "OP1",                  x = x, y = inc.y(tableSpacing.header) }
+fields[#fields + 1] = {                             x = x, y = inc.y(tableSpacing.row), min = 0, max = 1, vals = { val_1, val_2 }, table = { [0] = "OFF", "ON" } }
+
+x = x + tableSpacing.col
+y = yMinLim - tableSpacing.header
+
+labels[#labels + 1] = { t = "",                     x = x, y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "",                     x = x, y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "",                     x = x, y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "",                     x = x, y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "OP2",                  x = x, y = inc.y(tableSpacing.header) }
+fields[#fields + 1] = {                             x = x, y = inc.y(tableSpacing.row), min = 0, max = 1, vals = { val_1, val_2 }, table = { [0] = "OFF", "ON" } }
+
+x = x + tableSpacing.col
+y = yMinLim - tableSpacing.header
+
+labels[#labels + 1] = { t = "",                     x = x, y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "",                     x = x, y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "",                     x = x, y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "",                     x = x, y = inc.y(tableSpacing.header) }
+labels[#labels + 1] = { t = "OP3",                  x = x, y = inc.y(tableSpacing.header) }
+fields[#fields + 1] = {                             x = x, y = inc.y(tableSpacing.row), min = 0, max = 1, vals = { val_1, val_2 }, table = { [0] = "OFF", "ON" } }
+
+return {
+    read        = 84, -- MSP_OSD_CONFIG
+    write       = 85, -- MSP_SET_OSD_CONFIG
+    title       = "OSD Pos/Profil",
+    reboot      = false,
+    eepromWrite = true,
+    minBytes    = 4,
+    labels      = labels,
+    fields      = fields,
+    postLoad = function(self)
+        self.fields[1].value = address
+        self.fields[2].value = 1
+        self.fields[3].value = bit32.lshift(self.values[val_1], (1-1)*8)+bit32.lshift(self.values[val_2], (2-1)*8)
```

Comment:
Please give operators some room for readability:
```suggestion
        self.fields[3].value = bit32.lshift(self.values[val_1], (1 - 1) * 8) + bit32.lshift(self.values[val_2], (2 - 1) * 8)
```